### ISSUE TITLE
The generator is not working for Rails 5.1.4

### ISSUE THREAD

**Issue by RailsCod3rFuture** [TrestleAdmin/trestle-auth#9]
I'm receiving the message below. Neither the migration nor the initializer is being created.
```
C:\Users\D\RubymineProjects\TestApp>rails g trestle:auth:install
      insert  config/initializers/trestle.rb
The file C:/Users/D/RubymineProjects/TestApp/config/initializers/trestle.rb does not appear to exist

```

**Comment by spohlenz** [comment]
It looks like you may not have run the original Trestle install generator (`rails g trestle:install`) before running the trestle-auth generator (please correct me if I'm wrong).

There should definitely be a nicer error message in this case though. I'll leave this issue open until that is in place.

**Comment by spohlenz** [comment]
Fixed in 4913637.

**Comment by hdbreaker** [comment]
Hi rails g trestle:install get stuck and never end.
